link

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env php
+<?php
+
+/*
+* This file is part of the Symfony package.
+*
+* (c) Fabien Potencier <fabien@symfony.com>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+/**
+ * Link dependencies to components to a local clone of the monolithic Symfony repository.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+
+if ($argc < 2) {
+    echo 'Link dependencies to components to a local clone of the monolithic Symfony repository.'.PHP_EOL.PHP_EOL;
+    echo 'Usage: ./link /path/to/the/project'.PHP_EOL;
+    exit(1);
+}
+
+if (!is_dir("$argv[1]/vendor/symfony")) {
+    echo "The directory \"$argv[1]\" does not exist or the dependencies are not installed, did you forget to run \"composer install\" in your project?";
```

Comment:
missing `PHP_EOL`?

given `The directory "foo" does not exist or the dependencies are not installed, did you forget to run "composer install" in your project?user@docker:/some/path$`